pop3/locker/locker.go

```diff
@@ -0,0 +1,21 @@
+package locker
```

Comment:
Is a separate package really needed? I know I separated it from code in `gomaild`, but now only POP3 uses it...